src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -54,9 +54,40 @@ public void WriteClient(CodeWriter writer, LowLevelClient client)
                     WriteClientFields(writer, client);
                     WriteConstructors(writer, client);
 
+                    HashSet<LowLevelClientMethod> visitedMethods = new HashSet<LowLevelClientMethod>();
                     var exampleComposer = new LowLevelExampleComposer(client);
+                    foreach (var convenienceMethod in client.ConvenienceMethods)
```

Comment:
Why not traverse the client.ClientMethods, and in each loop, check if it is need to generate convenience method, if so, just add one?
Write in this way may not cause too many duplicate code.